src/svgcontext.ts

```diff
@@ -113,18 +188,20 @@ export class SVGContext implements RenderContext {
     };
 
     this.state_stack = [];
-
-    // Test for Internet Explorer
-    this.iePolyfill();
   }
 
+  create(svgElementType: 'g'): SVGGElement;
+  create(svgElementType: 'path'): SVGPathElement;
+  create(svgElementType: 'rect'): SVGRectElement;
+  create(svgElementType: 'svg'): SVGSVGElement;
+  create(svgElementType: 'text'): SVGTextElement;
   create(svgElementType: string): SVGElement {
     return document.createElementNS(SVG_NS, svgElementType);
   }
```

Comment:
Hey, I learned something new (overload signatures)!

https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature

Do you think others might be confused that the last signature `create(svgElementType:string)` cannot be used directly?

Maybe we need a comment that create() can only be called with 5 different parameters: svgElementType = 'g' | 'path' | ....

Reply:
Oops, I forgot to add the generic overload. Passing an arbitrary string to `create` will compile now.